ingestion/tests/integration/orm_profiler/test_orm_profiler_e2e.py

```diff
@@ -241,7 +240,7 @@ def test_profiler_workflow(self):
         ).profile
 
         assert not table.tableProfilerConfig
```

Comment:
Can you count the rows from the profile as well?

Reply:
Resolved

Reply:
line 246 `test_workflow_sample_profile` definition

Reply:
yep, on it.

Reply:
Done